notebooks/data-sources/TestGrid/metrics/number_of_flakes.ipynb

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
<p>this would be "saving only first 1000000 out of.."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>This cell gives me an error since the directory does not exist already. to_csv creates the file but not the directory and sub-directory. </p><pre class="ql-syntax" spellcheck="false">FileNotFoundError: [Errno 2] No such file or directory: '../../../../data/processed/metrics/percent_flake_by_test/flakes.csv'
</pre><p> </p><p>To overcome that, we can explicitly create the directories</p><p> </p><p><code></code><code>python</p><p>file = 'flakes.csv'</p><p> </p><p>folder = '../../../../data/processed/metrics/percent_flake_by_test'</p><p>if not os.path.exists(folder):</p><p>&nbsp;&nbsp;os.makedirs(folder)</p><p> </p><p>fullpath = os.path.join(folder, file)&nbsp;&nbsp;&nbsp;</p><p></code><code></code></p>